modules/http-api/src/main/java/org/mule/runtime/http/api/sse/SseRetryConfig.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.sse;
+
+/**
+ * The {@link ServerSentEventSource} implements the corresponding
+ * <a href="https://html.spec.whatwg.org/multipage/server-sent-events.html">server-sent-events spec</a>. This interface allows
+ * configuring the retry mechanism described there.
+ */
+public interface SseRetryConfig {
```

Comment:
didn't we said this was going to be an immutable pojo? (or it could even be a record now)

Reply:
Changed to a record